tests/nanoemoji_test.py

```diff
@@ -42,7 +44,16 @@ def _run(cmd, tmp_dir=None):
         str(tmp_dir),
     ) + tuple(str(c) for c in cmd)
     print("subprocess:", " ".join(cmd))  # very useful on failure
-    subprocess.run(cmd, check=True)
+    subprocess.run(
+        cmd,
+        check=True,
+        env={
```

Comment:
I think by default (`env=None`) subprocess will inherit the whole `os.environ`. Since you are only modifying the `PATH` and `PYTHONPATH` variables, I would suggest that you first make a os.environ.copy(), update that in-place with your own variables and pass it on to subprocess. That way you don't need to list all the variables that may be needed (like you do for SYSTEMROOT on windows for example), the whole environment is passed through

Reply:
I deliberately didn't do that because I don't want random outside environment variables interfering.

Reply:
Resolving but plz reopen if you think this is a ludicrously bad idea